app_dart/lib/src/request_handlers/postsubmit_luci_subscription.dart

```diff
@@ -52,22 +51,6 @@ class PostsubmitLuciSubscription extends SubscriptionHandler {
       return Body.empty;
     }
 
-    if (buildPushMessage.userData.containsKey('repo_owner') && buildPushMessage.userData.containsKey('repo_name')) {
```

Comment:
For my understanding, could you explain what the removal of this section does?

Reply:
This seems to be a duplicate check of below. We already have the commit slug below via commit_key.